discovery/seata-discovery-core/src/main/java/io/seata/discovery/registry/RegistryService.java

```diff
@@ -88,4 +88,12 @@
      * @throws Exception
      */
     void close() throws Exception;
+
+    /**
+     * Get current service group name
+     *
+     * @param key service group
+     * @return the service group name
+     */
+    String getServiceGroup(String key);
```

Comment:
I think you should define a default method.

Reply:
Had replaced.